Makefile

```diff
@@ -24,6 +24,10 @@ AUTHORS: .mailmap .git/HEAD
 version/version.go:
 	./version/version.sh > $@
 
+${PREFIX}/bin/garbagecollect: version/version.go $(shell find . -type f -name '*.go')
```

Comment:
Not sure what you mean by this, can you clarify? Just following how its done for other items under cmd/